spec/models/miq_policy_spec.rb

```diff
@@ -307,4 +307,25 @@
       )
     end
   end
+
+  context '.validates' do
+    it 'validates towhat and mode' do
+      expect(described_class.create!(:towhat      => "Host",
+                                     :mode        => "compliance",
+                                     :active      => false,
+                                     :description => 'x',)).to have_attributes(:towhat => "Host",
+                                                                               :active => false,
+                                                                               :mode   => "compliance")
+    end
+
+    it 'reports invalid towhat' do
+      expect do
+        described_class.create!(:towhat      => "BobsYourUncle",
```

Comment:
Similar here

```ruby
expect(FactoryGirl.build(:miq_policy, :towhat => "BobsYourUncle")).not_to be_valid
```

Reply:
@bdunne Here again, if you feel strongly I'd be glad to change this test but for me it's much easier to quickly grasp what the test is doing, that a particular exception is expected and it's more consistent with existing tests in the spec.

Let me know if you are OK with it  as is. 

Thank you for the review feedback.
JoeV

Reply:
I'd be ok with testing `create!` if you're concerned with a custom validation message but since this is out of the box validation messages, I think you can just do `thing.new` + `thing.valid?` and maybe even `thing.errors[: towhat].blank?` or something like that.

Reply:
@jrafanie Thank you for explaining. Will do.

JoeV